.cirun.yml

```diff
@@ -0,0 +1,22 @@
+# Self-Hosted Github Action Runners on GCP via Cirun.io
+# Reference: https://docs.cirun.io/reference/yaml.html
+runners:
+  - name: gpu-runner
+    # Cloud Provider: GCP
+    cloud: gcp
+    # Cheapest GPU on GCP
+    gpu: nvidia-tesla-t4
+    # Cheapest VM on GCP, with GPU attachable
+    instance_type: n1-standard-1
+    # Ubuntu-20.4, can be seen from "gcloud compute images list"
+    machine_image: ubuntu-minimal-2004-lts
+    # preemptible instances seems quite less reliable.
+    preemptible: false
+    # Number of runners to provision on every trigger on Actions job
+    # 2 because testing on Python 3.7 & 3.8
+    # See .github/workflows/build-gpu.yml
+    count: 2
```

Comment:
Curious: if this was set to 1, would the tests effectively run sequentially?

Reply:
Yes, unless the machine gets killed for being idle for too long.